src/core/tsi/alts/handshaker/alts_tsi_handshaker.cc

```diff
@@ -533,9 +542,26 @@ static void alts_tsi_handshaker_create_channel(
   grpc_channel_credentials* creds = grpc_insecure_credentials_create();
   // Disable retries so that we quickly get a signal when the
   // handshake server is not reachable.
-  grpc_arg disable_retries_arg = grpc_channel_arg_integer_create(
-      const_cast<char*>(GRPC_ARG_ENABLE_RETRIES), 0);
-  grpc_channel_args args = {1, &disable_retries_arg};
+  std::vector<grpc_arg> args_vec;
+  args_vec.push_back(grpc_channel_arg_integer_create(
+      const_cast<char*>(GRPC_ARG_ENABLE_RETRIES), 0));
+  // TODO(gtcooke94) - Flag to try new values for ALTS keep alive settings,
+  // remove after trial
+  std::optional<std::string> env =
+      grpc_core::GetEnv(GRPC_EXPERIMENTAL_ALTS_HANDSHAKER_KEEPALIVE_PARAMS);
+  if (env.has_value()) {
+    bool use_experimental_params = false;
+    std::istringstream(*env) >> std::boolalpha >> use_experimental_params;
```

Comment:
Why is this line needed?

Reply:
It will make string `true/false` be interpreted as boolean `true/false` - since this is coming from an env arg I think this is what we want?

Reply:
Do you think we should make it very constrained in terms of what values we accept? For example:

```
bool use_experimental_params = (*env == "true");
```

Reply:
I'm fine with that